src/Randomizer.SMZ3.Tracking/VoiceCommands/CheatsModule.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Randomizer.Shared;
+using Randomizer.SMZ3.Regions.Zelda;
+using Randomizer.SMZ3.Tracking.AutoTracking;
+using Randomizer.SMZ3.Tracking.Configuration;
+
+namespace Randomizer.SMZ3.Tracking.VoiceCommands
+{
+    /// <summary>
+    /// Module for cheats via the auto tracker
+    /// </summary>
+    public class CheatsModule : TrackerModule
+    {
+        private readonly ILogger<AutoTrackerModule> _logger;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AutoTrackerModule"/>
+        /// class.
+        /// </summary>
+        /// <param name="tracker">The tracker instance.</param>
+        /// <param name="logger">Used to write logging information.</param>
+        public CheatsModule(Tracker tracker, ILogger<AutoTrackerModule> logger) : base(tracker, logger)
+        {
+            _logger = logger;
+
+            AddCommand("Heal player", HealPlayerRule(), (tracker, result) =>
+            {
+                tracker.GameInteractor?.HealPlayer();
+            });
+
+            AddCommand("Give item", GiveItemRule(), (tracker, result) =>
+            {
+                var item = GetItemFromResult(tracker, result, out var itemName);
+                tracker.GameInteractor?.GiveItem(item);
```

Comment:
We could introduce the random poll chance here too. Greater chances for "better" items?

Plenty of opportunity for sass, too. "I want it back by next seed", etc.